tools/common.cmd

```diff
@@ -0,0 +1,4 @@
+set configdir=%LOCALAPPDATA%\darktable
+set cache_base=%LOCALAPPDATA%\Microsoft\Windows\INetCache\darktable
```

Comment:
Win 7 is `Temporary Internet Files` instead of `INetCache`.  Works for Win 10 and Win 11